### ISSUE TITLE
Avoid high frequent changes in DPPO

### ISSUE THREAD

**Issue by dynamik1703** [MorvanZhou/Reinforcement-learning-with-tensorflow#34]
Hey MorvanZhou,

thank you for your really great work! I like your implementations a lot.

I have a question about the DPPO Algorithm.

In my own environments, it tends to do high frequent changes between the min and max action. Is there a way to avoid it?

Best,
Roman

**Comment by MorvanZhou** [comment]
Try to lower the learning rate first. The high learning rate may be the reason.

**Comment by dynamik1703** [comment]
Indeed, thanks!

You're really an expert!
